From 6276a48eca296b0e98c829f9911ce22e14d7224f Mon Sep 17 00:00:00 2001
From: Martin Erik Werner <martinerikwerner@gmail.com>
Date: Fri, 3 Apr 2015 22:04:45 +0200
Subject: [PATCH 4/5] appdata: Update description & screenshots

* Match readme description, but add "cross-platform multiplayer" instead
  of platform list
* Pick nicest (my arbitrary opionion) three screenshots and add captions
---
 src/install/nix/redeclipse.appdata.xml | 27 +++++++++++++++------------
 1 file changed, 15 insertions(+), 12 deletions(-)

diff --git a/src/install/nix/redeclipse.appdata.xml b/src/install/nix/redeclipse.appdata.xml
index 30a642b..60b9e2f 100644
--- a/src/install/nix/redeclipse.appdata.xml
+++ b/src/install/nix/redeclipse.appdata.xml
@@ -6,27 +6,30 @@
   <description>
     <p>
       Red Eclipse is a fun-filled new take on the casual first person arena
-      shooter with a general theme of agility in a variety of environments.
-    </p>
-    <p>
-      Red Eclipse features parkour, impulse boosts, dashing, other tricks,
-      and favourite game modes with an array of mutators and variables.
-      A builtin editor lets you create your own maps cooperatively online.
+      shooter, built as a total conversion of Cube Engine 2, which lends itself
+      toward a balanced gameplay, with a general theme of agility in a variety
+      of environments.
     </p>
+    <ul>
+      <li>Cross-platform multiplayer</li>
+      <li>Features parkour, impulse boosts, dashing, and other tricks</li>
+      <li>Favourite gamemodes with an array of mutators and variables</li>
+      <li>Builtin editor lets you create your own maps cooperatively online</li>
+    </ul>
   </description>
   <url type="homepage">http://redeclipse.net/</url>
   <screenshots>
     <screenshot type="default">
-      <image>http://redeclipse.net/bits/images/003.jpg</image>
-      <caption><!-- Describe this screenshot in less than ~10 words --></caption>
+      <image>http://redeclipse.net/bits/images/053.jpg</image>
+      <caption>Posing with fire on the map venus</caption>
     </screenshot>
     <screenshot>
-      <image>http://redeclipse.net/bits/images/006.jpg</image>
-      <caption><!-- Describe this screenshot in less than ~10 words --></caption>
+      <image>http://redeclipse.net/bits/images/045.jpg</image>
+      <caption>Mine fireworks on the map vault</caption>
     </screenshot>
     <screenshot>
-      <image>http://redeclipse.net/bits/images/007.jpg</image>
-      <caption><!-- Describe this screenshot in less than ~10 words --></caption>
+      <image>http://redeclipse.net/bits/images/019.jpg</image>
+      <caption>Overlooking friendly fire on the map cutec</caption>
     </screenshot>
   </screenshots>
   <updatecontact>http://redeclipse.net/forum/</updatecontact>
-- 
2.1.4

